assets/Gruntfile.js

```diff
@@ -199,7 +199,21 @@ module.exports = function (grunt) {
           'bower_components/messenger/build/css/messenger-theme-air.css',
           'bower_components/messenger/build/css/messenger-theme-ice.css',
           'bower_components/messenger/build/js/messenger-theme-future.js',
-          'bower_components/fontawesome/css/font-awesome.css'
+          'bower_components/fontawesome/css/font-awesome.css',
+          "bower_components/patternfly/dist/js/patternfly.js",
```

Comment:
I think we should. When I was testing the utilization bar chart, leaving it off caused a runtime error.